packages/theme/src/theme/_modals.scss

```diff
@@ -1,36 +1,40 @@
-
 .modal-dialog {
 	max-height: 80vh;
 	display: flex;
 	flex-direction: column;
 }
 
 .modal {
-
+	&-content {
+		display: flex;
+		flex-direction: column;
+	}
 	&-header {
 		border-top-left-radius: $border-radius-base;
 		border-top-right-radius: $border-radius-base;
 		background-color: $modal-header-bg;
+		border-color: $concrete;
 
 		.modal-title {
-			font-weight: bold;
+			font-weight: 400;
+			font-size: 16px;
```

Comment:
There is no existing variable for that font-size? 🤔

Reply:
$form-input-font-size: 16px !default; :/

Reply:
ok forget it :(